source/_integrations/weather.markdown

```diff
@@ -193,3 +193,99 @@ weather.toronto_forecast:
 ```
 
 {% enddetails %}
+
+## Service `weather.get_forecast_attribute`
+
+This service populates [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data)
+with a mapping of weather services and their associated forecasts.
```

Comment:
This is just copied from the `weather.get_forecast` service, without adjusting the language to what the new service does.